src/Libraries/RevitNodes/Elements/InternalUtilities/ParseEnum.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Autodesk.DesignScript.Runtime;
+
+namespace Revit.Elements
+{
+    /// <summary>
+    /// Parse Revit Enum
+    /// </summary>
+    [IsVisibleInDynamoLibrary(false)]
+    public class ParseEnum
+    {
+        /// <summary>
+        /// Parse any Revit Enum by String
+        /// </summary>
+        /// <param name="value">enum string value</param>
+        /// <param name="typeName">full type name</param>
+        /// <returns></returns>
+        [IsVisibleInDynamoLibrary(false)]
+        public static object ByString(string value, string typeName)
```

Comment:
usually I would implement it using a generic Type, 
public static T ByString<T>(...) where T : ...
But since I'm calling this using AstFactory.BuildStringNode, I guess it won't work.